### ISSUE TITLE
says it installs but wont boot to home brew menu 

### ISSUE THREAD

**Issue by jak8oflegend** [yellows8/3ds_homemenuhax#41]
i install 
it installs or claims to 
i reboot hold L boots to main menu

**Comment by yellows8** [comment]
Did you use the pad-config menu at all?

**Comment by jak8oflegend** [comment]
ive tried both ways  complete reformat and re hack  curently no config

**Comment by yellows8** [comment]
System-version / etc?

**Comment by jak8oflegend** [comment]
system version 9.2.0-20u hacked using rxtools and fbi inject and browserhax

**Comment by jak8oflegend** [comment]
the tutorial i followed told me to install theme hax 1.3 first which didn't work either could that be the problem

**Comment by yellows8** [comment]
"the tutorial i followed told me to install theme hax 1.3 first " That tutorial is probably obsolete.

You're attempting to use menuhax v2.0 right?

**Comment by jak8oflegend** [comment]
yep

**Comment by yellows8** [comment]
Try deleting your theme-cache on SD: http://3dbrew.org/wiki/Extdata
Then enter Home Menu theme-settings so that extdata gets recreated, then try installing menuhax again.

**Comment by jak8oflegend** [comment]
same end result installs complains about bgm but i understand its supposed to do taht i reboot hold down l and boots to home menu

**Comment by yellows8** [comment]
Try with other PAD buttons after using the PAD config menu.

**Comment by jak8oflegend** [comment]
same thing

**Comment by yellows8** [comment]
Does Home Menu display any message if you enter theme-settings?

**Comment by jak8oflegend** [comment]
nothing pops up

**Comment by yellows8** [comment]
Maybe try formatting your SD FS?

**Comment by nfabia90** [comment]
I'm using 10.2.0-28 U

**Comment by ghost** [comment]
Menuhax fixed...

**Comment by mumei4** [comment]
Something similar is happening to me, I had the latest menuhax installed and it worked fine for a while. Holding L would boot straight to the Homebrew Launcher but i had a moment of stupidity and decided to change my theme. This has now caused menuhax to no longer boot when I hold L, ending up in the home menu instead. Changing back to the default theme didn't fix this problem so would there be a way i can fix this without having to reinstall?

**Comment by ghost** [comment]
If you've got a N3DS, try flashing browserhax QR, go to params-> internet -> your connection-> save->no test, exit. Open browser asap, spam refresh until it loads. Reinstall.

**Comment by mumei4** [comment]
@WoniMK7, I have an O3DS with system version 10.1.0-27E so I'm not able to run browserhax anymore. And as I don't have access to any other entry points i'm not able to reinstall menuhax. Cubic Ninja is a bit too pricey just to run homebrew so it looks like i'll have to wait and see if something else becomes available for my system version...

**Comment by WoniMK7** [comment]
Use smashbroshax.

**Comment by ghost** [comment]
@mumei4 Use oot3dhax

**Comment by mumei4** [comment]
@WoniMK7, I've got oot3d ordered and it should be here in a weeks time. I have an Action Replay Powersaves device so i should have no problem setting it up. Thank you for your patience with me : )

**Comment by teuf504** [comment]
I have the same problem.

I have an O3DS running version 10.3. I installed menuhax via browserhax with the starter kit. The text says Menuhax has been succesfully installed but when I hold L when booting it won't launch. I tried to change the PAD type but it never launch.

I deleted the exdata cache then go to Home Menu Then and set a random them and installed it again just like you said but it didn't change...

Edit: never mind, I changed my SD card for a new one and it works fine now
